SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialActorChannel.h

```diff
@@ -60,6 +62,34 @@ class SPATIALGDK_API USpatialActorChannel : public UActorChannel
 		return NetDriver->StaticComponentView->HasAuthority(EntityId, Info->SchemaComponents[SCHEMA_ClientRPC]);
 	}
 
+	FORCEINLINE bool IsOwnedByWorker() const
+	{
+		if (NetDriver->GetNetMode() != NM_Client)
```

Comment:
Weren't we going to remove this check from this function, and include it in the NetOwner usage directly instead?